resources/views/popover-column.blade.php

```diff
@@ -53,10 +51,12 @@ class="h-5 w-5 text-gray-500 dark:text-gray-400"
         @endif
     </div>
 
-    <div class="z-50 fi-popover-content w-[{{ $getPopOverMaxWidth }}px] border border-gray-100 rounded-lg shadow-lg bg-white dark:bg-gray-800"
-         x-cloak x-trap="open"
-         x-show="open"
-         x-anchor.{{ $getPlacement }}.offset.{{ $getOffset }}="$refs.button"
+    <div class="z-50 fi-popover-content w-[{{ $getPopOverMaxWidth }}px] border border-gray-100 rounded-lg shadow-lg bg-white dark:bg-gray-800 transition"
+         x-cloak
+         x-ref="panel"
+         x-transition:enter-start="opacity-0"
+         x-transition:leave-end="opacity-0"
+         x-float.placement.bottom-end.flip.teleport.offset="{ offset: 8 }"
```

Comment:
shouldn't you still use getPlacement and getOffice here?

Reply:
yes this is just a wip, to make sure it fix the main issue. if all good, I'll switch all other components (form,infolist) to use `x-float`

Reply:
sure thing, yeah it does fix the issue...